common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -111,7 +111,7 @@ class InteractForm extends Component<Props, State> {
 
           <div className="input-group-wrapper InteractForm-address-field col-sm-6">
             <AddressFieldFactory
-              withProps={({ currentTo, isValid, onChange }) => (
+              withProps={({ isValid, onChange }) => (
```

Comment:
They both use the same selector so im not sure how it would desync, if i were to use `currentTo` from the factory component then i'd have to hoist the `AddressFieldFactory` one level up, making it kind of unsightly. Decided this would be the cleaner approach for now until we can refactor `Contracts` to use redux state for everything.